qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -65,11 +65,11 @@ class DrivenControl:
     .. math::
 
         H(t) = \frac{1}{2}\left(\Omega(t) e^{i\phi(t)} \sigma_- +
-                                \Omega(t) e^{-i\phi(t)}\sigma_+\right) +
+                                \Omega^{*}(t) e^{-i\phi(t)}\sigma_+\right) +
```

Comment:
Omega is real here, the phase phi is treated separately

Reply:
Duh, correct.

Reply:
Force pushed to correct this, it's too minor for another commit.

Reply:
Note that since we squash merge into master it doesn't really matter how messy the history in the branch gets.